x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/20_search_application_put.yml

```diff
@@ -152,25 +163,43 @@ teardown:
         body:
           indices: [ "test-index3", "test-index4" ]
           template: null
+
   - match: { result: "updated" }
 
   - do:
-     search_application.get:
-       name: test-updated-search-application
-  - match: {
-    template: {
-      script: {
-        source: "{\n  \"query\": {\n    \"query_string\": {\n        \"query\": \"{{query_string}}\",\n        \"default_field\": \"{{default_field}}\"\n        }\n    }\n}\n",
-        params: {
-          query_string: "*",
-          default_field: "*"
-        },
-        lang: "mustache"
-      }
-    }
-  }
+      search_application.get:
+        name: test-updated-search-application
+
+  - match:
```

Comment:
Should we consider including a dictionary with the default template that we provide?

Reply:
We could. We can include an array of strings for `default_field` and a string for `query_string`. Changing the template implies changing the dictionary as well, but I think that's to be expected.

Sounds good?

Reply:
The one issue with an array of Strings is the existing trailing comma issue: https://github.com/elastic/elasticsearch/pull/95200 

As I think about it more - let's punt on changes, and add a dictionary when we update the default template for 8.9.0 to be a bit more sophisticated. WDYT?

Reply:
I'm ok with that - TBH I had not default templates in mind for this change 👍